job.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')

```

Comment:
Duplicate! Present in models folder as well

Reply:
What to do in this case sir, please suggest!

Reply:
should I change the variable name